lxd/storage/drivers/driver_cephfs.go

```diff
@@ -138,6 +144,15 @@ func (d *cephfs) Create() error {
 		return errors.New("cephfs.path must match the source")
 	}
 
+	return nil
+}
+
+// Create is called during pool creation and is effectively using an empty driver struct.
+// WARNING: The Create() function cannot rely on any of the struct attributes being set.
```

Comment:
@roosterfish is the warning true anymore now you're validating the config up front?